vita3k/modules/SceHttp/SceHttp.cpp

```diff
@@ -57,48 +114,445 @@ EXPORT(int, sceHttpCookieImport) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceHttpCreateConnection) {
-    return UNIMPLEMENTED();
-}
-
-EXPORT(int, sceHttpCreateConnectionWithURL) {
-    return UNIMPLEMENTED();
+EXPORT(SceInt, sceHttpCreateConnectionWithURL, SceInt tmplId, const char *url, SceBool enableKeepalive) {
+#ifdef TRACY_ENABLE
+    // --- Tracy logging --- START
+    bool _tracy_activation_state = config::is_tracy_advanced_profiling_active_for_module(emuenv.cfg.tracy_advanced_profiling_modules, tracy_module_name);
+    ZoneNamedN(___tracy_scoped_zone, "sceHttpCreateConnectionWithURL", _tracy_activation_state);
+    tracy_sceHttpCreateConnectionWithURL(&___tracy_scoped_zone, _tracy_activation_state, tmplId, url, enableKeepalive);
+    // --- Tracy logging --- END
+#endif
+    if (!emuenv.http.inited)
+        return RET_ERROR(SCE_HTTP_ERROR_BEFORE_INIT);
+
+    if (emuenv.http.templates.find(tmplId) == emuenv.http.templates.end())
+        return RET_ERROR(SCE_HTTP_ERROR_INVALID_ID);
+
+    if (!url)
+        return RET_ERROR(SCE_HTTP_ERROR_INVALID_URL);
+
+    auto urlStr = std::string(url);
+
+    auto tmpl = emuenv.http.templates.find(tmplId);
+
+    int connId = emuenv.http.next_conn;
+    emuenv.http.next_conn++;
+
+    net_utils::parsedUrl parsed;
+    auto parseRet = net_utils::parse_url(url, parsed);
+    if (parseRet != 0) {
+        switch (parseRet) {
+        case SCE_HTTP_ERROR_UNKNOWN_SCHEME: {
+            LOG_WARN("SCHEME IS: {}", parsed.scheme);
```

Comment:
Decided to leave this warn just to get a more general knowledge over time on what games use the missing file scheme, it may take a while before i figure out how its supposed to work tho, so imo its better to just leave this warn here until it gets added, aside from that i think its ready for merge